packages/apps/desktop-viewer-test/src/frontend/components/App.tsx

```diff
@@ -85,12 +86,30 @@ const App = () => {
     <ThemeProvider theme="dark" style={{ height: "100%" }}>
       <SettingsContext.Provider value={{ settings, addRecent }}>
         <BrowserRouter>
-          <Routes>
-            <Route path="/" element={<HomeRoute />} />
-            <Route path="/itwins/:iTwinId" element={<IModelsRoute />} />
-            <Route path="/itwins" element={<ITwinsRoute />} />
-            <Route path="/viewer" element={<ViewerRoute />} />
-          </Routes>
+          <PageLayout>
+            <Routes>
+              <Route
+                element={
+                  <PageLayout.Content padded>
+                    <Outlet />
+                  </PageLayout.Content>
+                }
+              >
+                <Route path="/" element={<HomeRoute />} />
+                <Route path="/itwins/:iTwinId" element={<IModelsRoute />} />
+                <Route path="/itwins" element={<ITwinsRoute />} />
+              </Route>
+              <Route
+                element={
+                  <PageLayout.Content>
+                    <Outlet />
```

Comment:
Unfortunately react-router doesn't allow you to wrap individual Routes with something so this is the only way I can think of.
https://reactrouter.com/en/main/start/concepts#layout-routes